UWP/UWPHelpers/StorageExtensions.h

```diff
@@ -0,0 +1,93 @@
+// UWP STORAGE MANAGER
+// Copyright (c) 2023 Bashar Astifan.
+// Email: bashar@astifan.online
+// Telegram: @basharastifan
+// GitHub: https://github.com/basharast/UWP2Win32
+
+// Functions:
+// replace(std::string& str, const std::string& from, const std::string& to)
+// replace2(const std::string str, const std::string& from, const std::string& to)
+// split(const std::string s, char seperator)
+// 
+// isChild(std::string parent, std::string child)
+// isParent(std::string parent, std::string child, std::string childName)
```

Comment:
I don't really understand why you repeat the function declarations in comments - they'll just quickly drift out of date when changes are made.